source/features/linkify-code.tsx

```diff
@@ -42,8 +42,15 @@ void features.add(import.meta.url, {
 	],
 	exclude: [
 		pageDetect.isGist,
+		pageDetect.isPRFiles,
 	],
 	init,
+}, {
+	include: [
+		pageDetect.isPRFiles,
+	],
+	deduplicate: 'has-rgh-inner',
+	init,
```

Comment:
Even if there's a deduplicating class in the observer, we don't want to create unnecessary observers in the first place